### ISSUE TITLE
Better documentation for ssl mode in cli

### ISSUE THREAD

**Issue by pierreozoux** [owncloud/mail#1516]
It would be nice if the help for `mail:account:create` in the cli would display the possible options for `ssl-mode`. Something like that would be really helpfull:

```
ssl-mode for imap and smtp can accept the following values:
  `none`
  `ssl`: SSL/TLS
  `tls`: STARTTLS
```

I had to look the source code [here](https://github.com/owncloud/mail/blob/40d8f630c070c3347319788ab6216feaa632b3c8/js/templates/setup.html#L49-L51) to debug.

**Comment by ChristophWurst** [comment]
Yes, that can be a PITA. However, UX on the CLI has low priority (for me) as this command is meant for admins only.

Now that you know how to use the command, could you add this to our readme please? :D

**Comment by ChristophWurst** [comment]
Or even better, start what we discussed [here](https://github.com/owncloud/mail/issues/1164) and create something like ``doc/add-account-via-occ.md`` ;-)
